src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticOptions.cs

```diff
@@ -0,0 +1,42 @@
+#if !NET45
+using System;
+using System.Collections.Generic;
+using Datadog.Trace.Abstractions;
+using Microsoft.AspNetCore.Http;
+
+namespace Datadog.Trace.DiagnosticListeners
+{
+    internal class AspNetCoreDiagnosticOptions
+    {
+        private List<Func<HttpContext, bool>> _ignorePatterns;
+
+        /// <summary>
+        /// Gets a list of delegates that define whether or not a given request should be ignored.
+        /// </summary>
+        public List<Func<HttpContext, bool>> IgnorePatterns
```

Comment:
What is the purpose of this list of delegates for ignoring requests? Does it implement something that's done in the Tracer in previous implementation, or is it something new for this implementation?

Reply:
This is new. It came from [the implementation](https://github.com/opentracing-contrib/csharp-netcore/blob/c219e2a3ec6fd5c16f42545fdaa3955da3f05c4f/src/OpenTracing.Contrib.NetCore/AspNetCore/HostingOptions.cs#L32) that I based this changes on. It's not exposed to users yet, but that's a possibility in the future.